string info
uint64 time_stamp
uint16[3] distance
int16 servo_control
uint32[2] wheel_count
uint8[230400] camera_raw
